plugin/trino-hive/src/main/java/io/trino/plugin/hive/s3select/csv/S3SelectCsvRecordReader.java

```diff
@@ -61,16 +56,10 @@ public S3SelectCsvRecordReader(
     }
 
     @Override
-    public SelectObjectContentRequest buildSelectObjectRequest(Properties schema, String query, Path path)
+    public InputSerialization buildInputSerialization()
```

Comment:
Split the refactoring of the code in hindsight of code reuse in a preparatory commit.

Reply:
I don't follow the explanation.
Please reconsider splitting the PR in multiple commits.

Reply:
While I understand and appreciate your point of view, I don't find this a mandatory step for this PR, given that most of the changes are in test clases.